### PR TITLE
cliff2

### PR DESCRIPTION
**Pull request by yeelali14**
[workerB](https://workerb.linearb.io/v2/badge/collaboration-page?magicLinkId=sxgp9_O)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,3 +67,124 @@ Before it gets any better
...
+I scraped my knees while I was praying
+And found a demon in my safest haven, seems like
+It's getting harder to believe in anything
+Then just to get lost in all my selfish thoughts
+I wanna know what it'd be like
+To find perfection in my pride
+To see nothing in the light
+I'll turn it off
+In all my spite
+In all my spite
```
Comment (vim-zz) [inline README.md:80]: OMG!

**Review by vim-zz — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -67,3 +67,124 @@ Before it gets any better
...
+I'll turn it off
+
+I scraped my knees while I was praying
+And found a demon in my safest haven, seems like
+It's getting harder to believe in anything
+Then just to get lost in all my selfish thoughts
+I wanna know what it'd be like
+To find perfection in my pride
+To see nothing in the light
+I'll turn it off
```
Comment (vim-zz) [inline README.md:90]: number 2

**Review by vim-zz — APPROVED** [review]
2

File: README.md [hunk 3]
```diff
@@ -67,3 +67,124 @@ Before it gets any better
 We're headed for a cliff
 And in the free fall I will realize
 I'm better off when I hit the bottom
+
+I scraped my knees while I was praying
+And found a demon in my safest haven, seems like
+It's getting harder to believe in anything
+Then just to get lost in all my selfish thoughts
```
Comment (vim-zz) [inline README.md:74]: 3

**Review by vim-zz — APPROVED** [review]
3

**Review by EladKohavi — APPROVED** [review]
bla

**Pull request closed without merging**